automation-tests/test/e2etemp/TempProject/authFile.json

```diff
@@ -0,0 +1,16 @@
+{
+  "status": 0,
```

Comment:
This is an interesting file to include - is it necessary?

Reply:
No, that was a mistake, good catch, I'll go delete it.